Makefile

```diff
@@ -971,15 +971,9 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
-lint-md-clean:
-	$(RM) -r tools/remark-cli/node_modules
-	$(RM) -r tools/remark-preset-lint-node/node_modules
-
 lint-md-build:
-	if [ ! -d tools/remark-cli/node_modules ]; then \
-		cd tools/remark-cli && ../../$(NODE) ../../$(NPM) install; fi
-	if [ ! -d tools/remark-preset-lint-node/node_modules ]; then \
-		cd tools/remark-preset-lint-node && ../../$(NODE) ../../$(NPM) install; fi
+	if [ ! -d tools/remark-cli/node_modules/fsevents/lib ]; then \
```

Comment:
This will try to rebuild on non-macOS every time right? I _think_ fsevents is only needed on macOS, so this could be:

```bash
 	if [ "$(OSTYPE)" = darwin -a ! -d tools/remark-cli/node_modules/fsevents/lib ]; then \ 
```

Reply:
That seems to be right. I will replace this line with your comment.
+ https://github.com/strongloop/fsevents/blob/master/binding.gyp#L6

Reply:
I've replaced. CI also gets green :)